trace-dispatcher/doc/trace-dispatcher.md

```diff
@@ -555,6 +555,8 @@ message per second, then add this to your configuration file:
     maxFrequency: 1.0
 ```
 
+In Cardano a default configuration is given in the module Cardano.Node.Tracing.DefaultTraceConfig. In the config file all entries of the default configuration can be overridden. To remove a frequency limiter, define a limiter with maxFrequency 0.0.
```

Comment:
`Cardano.Node.Tracing.DefaultTraceConfig` is worth backticking.

Reply:
Okay